src/main/java/cz/jirutka/spring/exhandler/RestHandlerExceptionResolver.java

```diff
@@ -134,15 +136,19 @@ protected ModelAndView doResolveException(
         throw new NoExceptionHandlerFoundException();
     }
 
-    protected void processResponse(ResponseEntity<?> entity, NativeWebRequest webRequest) throws Exception {
+    protected void processResponse(ResponseEntity<?> entity, NativeWebRequest webRequest, Object handler) throws Exception {
 
         ModelAndViewContainer mavContainer = new ModelAndViewContainer();
+        MethodParameter returnType = null;
+        if (handler != null && handler instanceof HandlerMethod) {
+            returnType = ((HandlerMethod) handler).getReturnType();
+        }
```

Comment:
There should probably be an `else` block here that creates the `MethodParameter`/`returnType`in some other way, but I don't know how likely that scenario is. But as it stands, the code will crash if run with Spring MVC 4.2.0 and the if block is not run for some reason.